src/ops.jl

```diff
@@ -109,3 +109,13 @@ function _maxpool_with_inds(t::Tensor{T,M}, pdims::PoolDims{N,K,S,P,D};
 
   Tensor{T,M}(ptr[1], on(t)), Tensor{T,M}(ptr[2], on(t))
 end
+
+function _chunk(t::Tensor{T,N}, chunks=2, dims=1) where {T,N}
+  ts = [t.ptr for _ in 1:chunks]
+  op::Vector{Tensor{T,N}} = [t for _ in 1:chunks]
+  atg_chunk(ts, t.ptr, chunks, N - dims)
+  for i in 1:chunks
```

Comment:
This might just be stylistic, but initializing `op` that way feels unnecessary.
Something like 
```julia
[Tensor{T.N}(ts[i], on(t)) for i in 1:chunks]
```
Could suffice too, without an intermediate